components/vf-social-links/README.md

```diff
@@ -0,0 +1,51 @@
+# vf-social-links Component
+
+[![npm version](https://badge.fury.io/js/%40visual-framework%2Fvf-social-links.svg)](https://badge.fury.io/js/%40visual-framework%2Fvf-social-links)
+
+## Usage
+
+### Variants Available
+
+- `vf-social-links--outline` this removes the background and gives a border around each icon square.
+- `dark-mode` if this is insdie a component that has `dark-mode` theeme then the colours swap.
+
+The dos and don'ts of using this component.
```

Comment:
I think this line was a leftover of the boilerplate. 

Also a though that it would be good to make clear that for the icons to show you need to add the HTML snippet to the page:

```html
<svg display="none" class="vf-icon-collection vf-icon-collection--social">
  <defs>
```

Reply:
Or, rather, it needs to be present if you can't use the NJK template.

Reply:
> it would be good to make clear that for the icons to show you need to add the HTML snippet to the page

👍 

in lieu of working out a strategy for new vf/embl icons for this stuffs :D

Reply:
Yeah, really want to work on that one ... but then all those other things to do 😭